### ISSUE TITLE
Inconsistency from scala REPL

### ISSUE THREAD

**Issue by MayamaTakeshi** [com-lihaoyi/Ammonite#1189]
Hi,
this is not critical but it is suboptimal:
scala:
```
scala> val s = "おはようございます"  
s: String = おはようございます

scala> s.substring(0,1)
res0: String = お
```
amm: 
```
@ val s = "おはようございます"  
s: String = "\u304a\u306f\u3088\u3046\u3054\u3056\u3044\u307e\u3059"

@ s.substring(0,1) 
res1: String = "\u304a"
```

**Comment by lihaoyi** [comment]
should be fixed by https://github.com/com-lihaoyi/PPrint/pull/71

**Comment by lihaoyi** [comment]
Fixed in 2.5.0

```scala
Welcome to the Ammonite Repl 2.5.0 (Scala 2.13.7 Java 11.0.7)
@ val s = "おはようございます"
s: String = "おはようございます"
```
